library/include/ck/library/utility/host_tensor.hpp

```diff
@@ -440,6 +440,34 @@ struct Tensor
         return mData[mDesc.GetOffsetFromMultiIndex(idx)];
     }
 
+    Tensor<T> Transpose(std::vector<size_t> axes = {}) const
+    {
+        if(axes.empty())
+        {
+            axes.resize(this->GetNumOfDimension());
+            std::iota(axes.rbegin(), axes.rend(), 0);
```

Comment:
`std::iota` need `include <numeric>` ,  so please add explicit `include <numeric>`, at present, it depends on the implicit including in the examples.   I came across failures while compiling xformers C++ extension.  

So please check all other device operators files and have this issue fixed.